src/WebJobs.Script/Description/Binding/EventHubBindingMetadata.cs

```diff
@@ -11,6 +11,9 @@ public class EventHubBindingMetadata : BindingMetadata
         [AllowNameResolution]
         public string Path { get; set; }
 
+        // Optional Consume group
+        public string ConsumerGroup { get; set; }
```

Comment:
the json ToObject<EventHubBinding>() we have in place is what makes this work, right? If they use the right name in their binding metadata, this gets deserialized?